content/posts/2021/07-15-salt-nornir-intro.md

```diff
@@ -0,0 +1,229 @@
+# SALTSTACK Nornir Proxy Minion introduction
+
+Network Automation is a complex but interesting topic with a lot of problems to solve and a lot to learn, defined by one of the [vendors](https://www.cisco.com/c/en/us/solutions/automation/network-automation.html) as:
+
+> Network automation is the process of automating the configuring, managing, testing, deploying, and operating of physical and virtual devices within a network.
+
+This post is an introduction in what SALTSTACK is and what is Nornir and how Network Automation field can benefit from combining them together.
```

Comment:
I'd slightly rephrase:

"This post is an introduction to SALTSTACK and Nornir and how the Network Automation field can benefit from combining them"